Remove debug-test-runner, debug-minibrowser, and debug-safari scripts

sideshowbarker opened the pull request:
#### 061df64c0a97fbca7a74e54a10174f21483e5e67
<pre>
Remove debug-test-runner, debug-minibrowser, and debug-safari scripts
<a href="https://bugs.webkit.org/show_bug.cgi?id=268957">https://bugs.webkit.org/show_bug.cgi?id=268957</a>

Reviewed by Jonathan Bedard.

These scripts don’t actually work, and nobody seems to be using them,
and they’ve not been touched in more than 6 years.

* Tools/Scripts/debug-minibrowser: Removed.
* Tools/Scripts/debug-safari: Removed.
* Tools/Scripts/debug-test-runner: Removed.

Canonical link: <a href="https://commits.webkit.org/275486@main">https://commits.webkit.org/275486@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b1163518726a6f03d98a26b98b73396ab20ecea3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/38521 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/17455 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/40852 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/41062 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/34205 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/40827 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/20471 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/14800 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/32393 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/39094 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/14874 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/33505 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/12776 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/12889 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/34365 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/42338 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/35172 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/34765 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/38591 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/13368 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/11051 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/36796 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/14968 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/9402 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/13835 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/14441 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

gsnedders reviewed (commented):
[non-reviewer]

Do we have any idea how long these haven't worked for? https://bugs.webkit.org/show_bug.cgi?id=209113 suggests it's a regression since 2020.

I'm not against removing them, but the shorter they've been broken for the more surprising it might be to occasional contributors—we could just make them invoke `run-minibrowser --debug` etc instead.

sideshowbarker committed:
Remove debug-test-runner, debug-minibrowser, and debug-safari scripts
https://bugs.webkit.org/show_bug.cgi?id=268957

Reviewed by Jonathan Bedard.

These scripts don’t actually work, and nobody seems to be using them,
and they’ve not been touched in more than 6 years.

* Tools/Scripts/debug-minibrowser: Removed.
* Tools/Scripts/debug-safari: Removed.
* Tools/Scripts/debug-test-runner: Removed.

Canonical link: https://commits.webkit.org/275486@main
